src/node/internal/internal_utils.ts

```diff
@@ -200,3 +208,167 @@ export function getOwnNonIndexProperties(
   }
   return result;
 }
+
+function callbackifyOnRejected(
+  reason: unknown,
+  cb: (error?: unknown) => void
+): void {
+  if (!reason) {
+    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+    cb(new ERR_FALSY_VALUE_REJECTION(`${reason}`));
```

Comment:
would `String(reason)` allow removing the comment?

Reply:
yes. i personally think we should just remove this linting rule in the future.